mailpoet/mailpoet
# Remove mailpoet cron trigger [MAILPOET-4271]

Fixes [MAILPOET-4271]

[MAILPOET-4271]: https://mailpoet.atlassian.net/browse/MAILPOET-4271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Testing instructions:
1. Go to MailPoet > Settings > Advanced tab
2. Make sure you don't see anymore "MailPoet’s own script" as an option to send emails
3. Make sure you see "Visitors to your website (recommended)" and "Server side cron (Linux cron)"
4. Make sure "Visitors to your website (recommended)" is checked and do some sending (newsletters, subscribing a form and receiving confirmation email" in order to verify that the sending is working as expected (sending generally)
